internal/printer/print-to-js.go

```diff
@@ -316,7 +316,11 @@ func render1(p *printer, n *Node, opts RenderOptions) {
 
 	// Tip! Comment this block out to debug expressions
 	if n.Expression {
-		if n.FirstChild == nil {
+		clean := ""
+		if n.FirstChild != nil {
+			clean = strings.TrimSpace(n.FirstChild.Data)
+		}
+		if n.FirstChild == nil || clean == "" {
```

Comment:
We now need to check for whitespace too as the parser doesn't remove them in expressions anymore.